addons/hr_timesheet/models/project.py

```diff
@@ -169,7 +189,8 @@ def _compute_encode_uom_in_days(self):
     def _compute_analytic_account_active(self):
         """ Overridden in sale_timesheet """
         for task in self:
-            task.analytic_account_active = task.project_id.analytic_account_id.active
+            # FIXME: perhaps check if the user is portal one then add the sudo ?
```

Comment:
if the user has read access to the task, using `compute_sudo` is probably sufficient for this, and it makes no sense to give more access to portal users than internal ones

Reply:
It has been corrected in the last pushed commits :)